src/YieldDistributor.sol

```diff
@@ -43,18 +44,20 @@ contract YieldDistributor is IYieldDistributor, Ownable2StepUpgradeable, VotingM
     address[] public queuedProjectsForAddition;
     /// @notice Array of projects queued for removal from the next cycle
     address[] public queuedProjectsForRemoval;
-    /// @notice The voting power allocated to projects by voters in the current cycle
+    /// @notice The voting power allocated to each project by voters in the current cycle
     uint256[] public projectDistributions;
     /// @notice The last block number in which a specified account cast a vote
     mapping(address => uint256) public accountLastVoted;
-    /// @notice The voting power allocated to projects by voters in the current cycle
+    /// @notice The voting power allocated to each project by a specific voter in the current cycle
     mapping(address => uint256[]) voterDistributions;
     /// @notice How much of the yield is divided equally among projects
     uint256 public yieldFixedSplitDivisor;
     /// @notice The address of the `ButteredBread` token contract
     ERC20VotesUpgradeable public BUTTERED_BREAD;
     /// @notice The block number before the last yield distribution
     uint256 public previousCycleStartingBlock;
+    /// @notice The interface for the `VotingStreakMultiplier` contract
+    IVotingStreakMultiplier public votingStreakMultiplier;
```

Comment:
We don't want to add individual multipliers like this because (1) we can't because we can't call the initialize function again and (2) it's redundant because we're already maintaining a list in `VotingMultipliers`. Instead we should update or extend the functionality of `VotingMultipliers` and `IMultiplier` so that the `castVoteWithMultipliers` iterates over the multipliers and calls a state-altering "update" function for any multiplier that implements that function. I think for the sake of simplicity, we can just generically update `IMultiplier` to include "update" in the interface so it's available for any multiplier and then the one's that don't need it can just no op